docs/reference-implementation/Brownfield-Considerations.md

```diff
@@ -0,0 +1,43 @@
+# Brownfield scenario considerations
+
+While the initial usage of the Azure Mission-Critical reference implementation often times include deploying it into empty Azure subscription(s) aka greenfields, is it not limited to that. This guide describes some of the most common scenarios when deploying the Azure Mission-Critical reference implementation into existing environments aka brownfield scenarios.
+
+## Existing networking infrastructure
+
+The Azure Mission-Critical online reference implementation maintains its own virtual networks and subnets. It is supposed to be deployed fully independent of any existing infrastructure. For environments with existing networking infrastructure or requirements for connectivity to existing services either on-premises or in another virtual network, it is recommended to use the [Azure Mission-Critical - Connected](https://github.com/Azure/Mission-Critical-Connected/) reference implementation.
+
+Considerations when using existing networking infrastructure:
+
+- Prefer using individual virtual networks and subnets for each solution rather than existing, shared ones. If interconnection is required evaluate the use of Virtual Network Peerings or Private Endpoints. This is recommended to avoid potential issues due to changes made to the shared infrastructure, noisy neighbor scenarios etc.
+- For shared networks do upfront planning to avoid overlapping IP address ranges or exhaustion.
```

Comment:
@hjscherer what do you mean? What's meant with "shared" is using a virtual network that was created outside of MC. I think this is important to note here that one should not use existing vnets and in case its required for valid reasons refer to the connected RI.

Reply:
Or are you looking for something a bit more like this:

> The Azure Mission-Critical online reference implementation deploys and uses its own virtual networks and subnets. It is supposed to be deployed fully independent of any existing infrastructure. For environments with existing networking infrastructure or requirements for connectivity to existing services either on-premises or in another virtual network, it is required to use the [Azure Mission-Critical - Connected](https://github.com/Azure/Mission-Critical-Connected/) reference implementation.<br />
>Potential use cases that might require using existing infrastructure are:
>- Connectivity from/to on-premises systems via Express Route or VPN
>- Connectivity from/to other Virtual Networks and Subnets via Virtual Network Peering
>- Mandatory use of pre-provisioned Virtual Networks and Subnets

Reply:
Removed.